changelogs/unreleased/gh-6092-invalid-listen-address-make-tarantool-blind.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/core
+
+* Fixed the fact that using of invalid listen address made
+  tarantool blind. Now in case of invalid listen address
+  tarantool still listen old one (gh-6092).
```

Comment:
Please, add an empty line in the end of the file. Git diff warns about that.

Reply:
Fixed